src/mbedit/mbedit_prog.c

```diff
@@ -5337,10 +5336,9 @@ int mbedit_tsminmax(int iping, int nping, int data_id, double *tsmin, double *ts
 	if (iping >= 0 && nbuff > iping && nping > 0 && iping + nping - 1 < nbuff) {
 		mbedit_tsvalue(iping, data_id, tsmin);
 		*tsmax = *tsmin;
-		// TODO(schwehr): Localize value and value2?
-		double value;
-		double value2;
 		for (int i = iping; i < iping + nping; i++) {
+		  double value;
+		  double value2;
```

Comment:
value2 should then be pushed inside the `if (data_id == MBEDIT_PLOT_ROLL)`

Reply:
Done.